jepostule/static/js/common/localStorage.js

```diff
@@ -0,0 +1,35 @@
+'use strict'
+
+document.addEventListener('DOMContentLoaded', function () {
+
+  useLocalStorage(document.querySelectorAll('[data-localstorage]'))
+
+})
+
+let useLocalStorage = inputs => {
+
+  if (!inputs.length) {
+    return
+  }
+
+  // Load from local storage.
+  Array.from(inputs).forEach(input => {
+    let value = localStorage.getItem(input.name)
+    if (value) {
+      input.value = value
+      input.dispatchEvent(new Event('change'), {'bubbles': true})
+    }
+  })
+
+  let saveToLocalStorage = (e) => {
+    let input = e.target
+    if (input.value) {
+      localStorage.setItem(input.name, input.value)
+    }
+  }
+
+  Array.from(inputs).forEach(el => {
+    el.addEventListener('blur', saveToLocalStorage)
```

Comment:
`blur` ? c'est quoi cet évènement ?
https://developer.mozilla.org/en-US/docs/Web/API/Element/blur_event

Haaaaaa trop bien ! Je me suis cassé la tête à le chercher.
Mais alors euh attends... Est-ce que l'évènement `blur` est déclenché lorsqu'on clique sur "étape suivante" ? J'avais eu beaucoup de mal à faire fonctionner correctement cet évènement, je suggère de bien tester avant de mettre ça en prod.

Reply:
Of course `blur` est *trigger* au `clic` sur "étape suivante" :)